content/en/setup/module-2/index.mdx

```diff
@@ -6,17 +6,17 @@ description: Forking and customizing Ocean Market (Frontend)
 featuredImage: images/creatures/mantaray/mantaray-full@2x.png
 ---
 
-# 🐟 Forking and customizing Ocean Market (Frontend)
+# 🐟 customized
```

Comment:
This title seems a bit short now

Reply:
Ahh! This was replaced by mistake! I had find and replace option open and replaced the whole line unknowingly. Let me fix this.